### PR TITLE
[VEN-1719]: remove governance contracts and use governance package

### PR DESCRIPTION
**Pull request by GitGuru7**
Removes :
- Governor Contracts
- Timelock Contracts
- Governor Interfaces
Resolves [VEN-1719]
## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

**Review by Debugger022 — APPROVED** [review]

**Comment by chechu** [comment]
* The commit removing contracts should be a breaking change
* I would create a folder `legacy-abis` in the root folder of the repo, which should be included in the npm package, containing the the abi files of the removed GovernorAlpha contracts
* I would deploy to testnet and mainnet the new VenusLens. We will have to start using it in the UI, and we should update the docs site with the new addresses

**Review by coreyar — APPROVED** [review]

**Comment by chechu** [comment]
> * The commit removing contracts should be a breaking change
> * I would create a folder `legacy-abis` in the root folder of the repo, which should be included in the npm package, containing the the abi files of the removed GovernorAlpha contracts
> * I would deploy to testnet and mainnet the new VenusLens. We will have to start using it in the UI, and we should update the docs site with the new addresses

* we don't need the `legacy-abis` folder, because we have added the legacy contracts in the `governance-contracts` repo: https://github.com/VenusProtocol/governance-contracts/tree/main/contracts/legacy
* we already have the tests for the governance contract in the new repo ([here](https://github.com/VenusProtocol/governance-contracts/tree/main/tests/Governance/GovernanceBravo)). So, I suppose we can remove from here the tests in the folder `tests/hardhat/Governance`. Wdyt?
* regarding the changes in the VenusLens contract, updating the functions used to get info from the GovernorAlpha (getGovReceipts, getGovProposals), we should check first if we need them

File: contracts/Lens/VenusLens.sol [hunk 1]
```diff
@@ -345,14 +347,14 @@ contract VenusLens is ExponentialNoError {
      * @return Array of governor receipts
      */
     function getGovReceipts(
```
Comment (chechu) [inline contracts/Lens/VenusLens.sol:349]: Remove this function

File: contracts/Lens/VenusLens.sol [hunk 2]
```diff
@@ -65,7 +65,7 @@ contract VenusLens is ExponentialNoError {
     struct GovReceipt {
```
Comment (chechu) [inline contracts/Lens/VenusLens.sol:65]: Remove this struct

File: contracts/Lens/VenusLens.sol [hunk 3]
```diff
@@ -399,7 +405,7 @@ contract VenusLens is ExponentialNoError {
      * @return GovProposal structs for provided proposal Ids
      */
     function getGovProposals(
```
Comment (chechu) [inline contracts/Lens/VenusLens.sol:407]: Remove this function

File: contracts/Lens/VenusLens.sol [hunk 4]
```diff
@@ -81,8 +81,10 @@ contract VenusLens is ExponentialNoError {
         uint endBlock;
         uint forVotes;
         uint againstVotes;
+        uint abstainVotes;
```
Comment (chechu) [inline contracts/Lens/VenusLens.sol:84]: Remove this struct

File: contracts/Lens/VenusLens.sol [hunk 5]
```diff
@@ -369,7 +371,7 @@ contract VenusLens is ExponentialNoError {
      * @param governor Governor address
      * @param proposalId Id of a proposal
      */
-    function setProposal(GovProposal memory res, GovernorAlpha governor, uint proposalId) internal view {
+    function setProposal(GovProposal memory res, GovernorBravoDelegate governor, uint proposalId) internal view {
```
Comment (chechu) [inline contracts/Lens/VenusLens.sol:374]: Remove this function

**Review by chechu — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Pull request merged**